.github/workflows/lint.yml

```diff
@@ -52,13 +52,7 @@ jobs:
         run: terraform fmt -check -recursive
         working-directory: terraform
 
-      - name: Validate PaaS
-        run: |
-          terraform init -backend=false
-          terraform validate -no-color
-        working-directory: terraform/paas
-
-      - name: Validate AKS
```

Comment:
Why have this name change?

Reply:
No particular reason other than we're only validating one thing now.